lib/isbn/cli.ex

```diff
@@ -0,0 +1,44 @@
+defmodule ISBN.CLI do
+  def main(argv) do
+    parse_args(argv)
+  end
+
+  def parse_args(argv) do
+    argv
+    |> OptionParser.parse(switches: [help: :boolean], aliases: [h: :help])
+    |> elem(1)
+    |> args_to_internal_representation()
+    |> process()
+  end
+
+  def args_to_internal_representation([isbn]) do
+    try do
+      isbn
+      |> String.to_integer()
```

Comment:
Instead of using a `try/rescue` to make sure that the isbn string can be converted to an integer with `String.to_integer`, we can delegate this check to the `OptionParser.parse` function by adding a new switch: `isb --isbn 1234567891234`.
Then the switches value can be something like: `OptionParser.parse(switches: [help: :boolean, isbn: :integer], aliases: [h: :help, i: :isbn])`
I think this will look a bit nicer